bfdd/bfdd_nb_config.c

```diff
@@ -860,10 +1066,15 @@ int bfdd_bfd_sessions_single_hop_desired_echo_transmission_interval_modify(
 int bfdd_bfd_sessions_single_hop_required_echo_receive_interval_modify(
 	struct nb_cb_modify_args *args)
 {
+	uint32_t echo_interval = yang_dnode_get_uint32(args->dnode, NULL);
 	struct bfd_session *bs;
 
 	switch (args->event) {
 	case NB_EV_VALIDATE:
+		if (echo_interval == 0)
+                        return NB_OK;
+		if (echo_interval < 10000 || echo_interval > 60000000)
+			return NB_ERR_VALIDATION;
 		break;
```

Comment:
ditto: yang change + separate commit for changes related to common bfd.

Reply:
you seem to have removed those changes.
if the behavior can be kept unchanged, it is ok to me.